tests/load/test_glueray.py

```diff
@@ -0,0 +1,96 @@
+import time
+import uuid
+from typing import Dict, Iterable
+
+import boto3
+import pytest
+
+
+@pytest.fixture(scope="session")
+def wrangler_zip_location(cloudformation_outputs: Dict[str, str]) -> str:
+    return cloudformation_outputs["AWSSDKforpandasZIPLocation"]
+
+
+@pytest.fixture(scope="session")
+def glue_job_role_arn(cloudformation_outputs: Dict[str, str]) -> str:
+    return cloudformation_outputs["GlueJobRoleArn"]
+
+
+@pytest.fixture(scope="session")
+def glue_ray_athena_workgroup_name(cloudformation_outputs: Dict[str, str]) -> str:
+    return cloudformation_outputs["GlueRayAthenaWorkgroupName"]
+
+
+@pytest.fixture(scope="function")
+def glue_job1(
+    path: str,
+    wrangler_zip_location: str,
+    glue_job_role_arn: str,
+    glue_ray_athena_workgroup_name: str,
+) -> Iterable[str]:
+    session = boto3.session.Session()
+
+    s3_client = session.client("s3")
+    glue_client = session.client("glue")
+
+    script_path = f"{path}script.py"
+
+    bucket, key = tuple(script_path[len("s3://") :].split("/", 1))
+    s3_client.upload_file(
+        "test_infra/glue_scripts/glue_example_1.py",
```

Comment:
Suggestion: do we want to parametrise the script name as well? This way we can use the same fixture for different scripts?

Reply:
That's a great idea. I had no idea fixtures support parameterization. Thanks for the tip!